model-test/src/test/java/hu/blackbelt/judo/meta/jsl/runtime/NameDuplicationDetectionTests.xtend

```diff
@@ -40,18 +40,16 @@ class NameDuplicationDetectionTests {
             type string String min-size:0 max-size:128;
 
             entity A {
-                relation B b opposite a;
-                field String b;
+            	field String b;
             }
 
             entity B {
-                relation A a opposite b;
+                relation A a opposite-add:b;
             }
 
         '''.parse => [
-            assertDuplicateMemberName("Duplicate member declaration: 'b'", JsldslPackage::eINSTANCE.entityFieldDeclaration)
-            assertDuplicateMemberName("Duplicate member declaration: 'b'", JsldslPackage::eINSTANCE.entityRelationDeclaration)
-        ]
+            	m | m.assertError(JsldslPackage::eINSTANCE.entityRelationOppositeInjected, JslDslValidator.DUPLICATE_MEMBER_NAME)
```

Comment:
This change means different test scenario! When testg for opposite required, create new tests for that!